common/source/docs/common-lua-scripts.rst

```diff
@@ -254,13 +286,58 @@ The GPS library provides access to information about the GPS's on the vehicle.
 
 - :code:`first_unconfigured_gps()` - Returns nil or the instance number of the first GPS that has not been fully configured. If all GPS's have been configured this returns 255 if all the GPS's have been configured.
 
-Notify
-~~~~~~
+GCS (gcs:)
+~~~~~~~~~~
+
+- :code:`send_text(severity, "text")` - Will send the text variable or constant string with message severity . Severity level is :
```

Comment:
Don't put `"text"`, that should just say message or text, no quotes, as that implies a string literal which doesn't make sense. The description is complicated as well, just say that text should be a string to send to the GCS. You don't need to distinguish between a constant one or one from a variable, that just implies complexity that doesn't exist.

Reply:
fixed